android/app/src/main/java/org/openbot/env/ControllerConfig.java

```diff
@@ -59,7 +60,14 @@ private void setBoolean(String name, boolean value) {
   }
 
   public String getVideoServerType() {
-    return get("video_server", "WEBRTC");
+    return get("video_server", "RTP");
+  }
+
+  public String[] getVideoServerAddress() {
+    String ip = get("ip", "172.217.22.14");
```

Comment:
I think we should have some neutral IP as default or just something like N/A if no server is setup.

Reply:
I changed it to "127.0.0.1"